README.md

```diff
@@ -44,7 +44,7 @@ Portainer CE is an open source project and is supported by the community. You ca
 Learn more about Portainers community support channels [here.](https://www.portainer.io/help_about)
 
 - Issues: https://github.com/portainer/portainer/issues
-- Slack (chat): https://portainer.io/slack/
+- Slack (chat): https://portainer.slack.com/
```

Comment:
@hookenz cool!

I think we should use the Slack link with the auto-invite/shared invite. Otherwise people will need to create a Slack account and get manually invited.

You can find the link on our website footer under "Connect".

Reply:
Good point.  I'll use that one.